devops/Makefile

```diff
@@ -7,6 +7,7 @@ export NAMESPACE := $(or $(NAMESPACE), $$NS)
 export APP_NAME := $(or $(PROJECT_NAME), sims)
 export TEMPLATE_FILE_NAME := $(or $(TEMPLATE_FILE_NAME), $$FILE)
 export BUILD_REF := $(or $(BUILD_REF), $(GIT_LOCAL_BRANCH))
+export DEV_NAMESPACE := $(or $(DEV_NAMESPACE), 0c27fb-dev)
```

Comment:
If the [below suggestion](https://github.com/bcgov/SIMS/pull/5470/files#r2594200456) is accepted, this could be removed.

Reply:
it think it should be okay to have the DEV_NAMESPACE defined as a global level